python/schedule.cpp

```diff
@@ -256,6 +268,8 @@ void register_schedules(py::module& m) {
             "The expected frequency [kHz].")
         .def_readwrite("seed", &poisson_schedule_shim::seed,
             "The seed for the random number generator.")
+        .def_property("tstop", &poisson_schedule_shim::get_tstop, &poisson_schedule_shim::set_tstop,
+            "No events delivered after this time [ms].")
```

Comment:
`std::ostream& operator<<(std::ostream& o, const poisson_schedule_shim& p)` (line 27 of this file) needs to be amended to print out the `tstop` time as well.

Reply:
Will do.